css/Style.css

```
@@ -353,57 +488,147 @@ button {
 }
 
 .footer-social {
-  width: 145px;
-  justify-self: center;
+  width: 150px;
   text-align: center;
-  outline: 5px solid rgba(0, 255, 0, 0.7);
-  outline-offset: -5px;
 }
 
 .footer-social ul {
   display: flex;
-  flex-wrap: wrap;
   justify-content: space-between;
   list-style: none;
-  width: 141px;
   margin: 0 auto;
   padding: 0;
 }
 
+.foter-social li {
 
-/* Social button */
+}
 
-.social-button {
+.footer-copyright {
   display: flex;
   align-items: center;
-  justify-content: center;
+  justify-content: space-around;
+  max-width: 350px;
+  margin: 0;
+  font-size: 14px;
+  line-height: 26px;
+}
+
+.footer-copyright .button:hover,
+.footer-copyright .button:focus {
+  color: var(--basic-black);
+  background-color: var(--basic-white);
+}
+
+
+/* Social buttons */
+
+.social-button li {
   width: 46px;
   height: 48px;
+  text-align: center;
+  margin: 0;
   color: inherit;
   background-color: var(--basic-blue);
 }
```

Предлагаю тут убрать ширину и высоту, будем формировать ее паддингами от ссылки. Это важно, для того, чтобы область клика была размером с весь блок, а не только размером с иконку (как сейчас)  

```suggestion
.social-button li {
  text-align: center;
  margin: 0;
  color: inherit;
  background-color: var(--basic-blue);
}
```

```
a {
   padding: 15px 15px 9px;
}
```

-----
или же можешь выровнять иконки проще, но менее доступно для пользователя:

```suggestion
.social-button li {
  width: 46px;
  height: 48px;
  text-align: center;
  margin: 0;
  display: flex;
  color: inherit;
  background-color: var(--basic-blue);
  justify-content: center;
  align-items: center;
}
```

---

выровнял паддингом, мне конечно больше нравится когда доступно для пользователя, но сами иконки далеки до идеала(